### PR TITLE
create new docker_run for existing cstar

### PR DESCRIPTION
**Pull request by polandll**
gh issue #32

### CODE REVIEW

File: modules/quickstart/partials/bash/docker_run_existing_cstar.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+=== Starting Stargate with existing Cassandra cluster
+
+Stargate can be added to an existing Cassandra cluster operating in a
+containerized environment with the following command:
+
+[source,bash,subs="attributes+"]
+----
+docker run --name stargate \
```
Comment (dougwettlaufer) [inline modules/quickstart/partials/bash/docker_run_existing_cstar.sh:8]: Should we call out that this isn't going to work on Mac due to docker networking and that if you want to you'll need either docker-compose or something like

```
docker network create cass-net --driver bridge

docker run --name some-cassandra \
--network cass-net \
-e CASSANDRA_CLUSTER_NAME=stargate-cluster \
-d cassandra:3.11

docker run --name stargate \
--network cass-net \
-p 8081:8081 \
-p 8082:8082 \
-p 9042:9042 \
-e CLUSTER_NAME=stargate-cluster \
-e CLUSTER_VERSION=3.11 \
-e SEED=some-cassandra  \
-e SIMPLE_SNITCH=true \
-e ENABLE_AUTH=true \
-d stargateio/stargate-3_11:v0.0.5
```

File: modules/quickstart/partials/bash/docker_run_existing_cstar.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  -d \
+  -e CLUSTER_NAME=stargate \
+  -e SEED=127.0.0.1 \
+  -e CLUSTER_VERSION=3.11 \
+  -e SIMPLE_SNITCH=true \
+  -e ENABLE_AUTH=true \
+  stargateio/stargate-3_11:{stargate-docker-tag}
+----
+
+where
```
Comment (dougwettlaufer) [inline modules/quickstart/partials/bash/docker_run_existing_cstar.sh:22]: Related to https://github.com/stargate/docs/issues/32#issuecomment-718781008 should we call out what's happening with `LISTEN_ADDRESS`? That if it's not provided then the start script will find it

File: modules/quickstart/partials/bash/docker_run_existing_cstar.sh (hunk 1 above)
Comment (pmehra7) [reply modules/quickstart/partials/bash/docker_run_existing_cstar.sh:8]: I think that would be worthwhile. There will be many developers using docker on OSX that may try this approach.

Comment (polandll) [reply modules/quickstart/partials/bash/docker_run_existing_cstar.sh]: Resolved this by adding example.

File: modules/quickstart/partials/bash/docker_run_existing_cstar.sh (hunk 2 above)
Comment (polandll) [reply modules/quickstart/partials/bash/docker_run_existing_cstar.sh]: Incorporated.

**Review by dougwettlaufer — APPROVED** [review]

**Pull request merged**